modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -263,36 +267,57 @@ const RichText = ({
 		>
 			<ClayInput.Group>
 				<ClayInput.GroupItem>
-					<ClassicEditor
-						ariaLabel={label}
-						ariaRequired={otherProps.required}
-						className="w-100"
-						contents={
-							currentValue
-								? currentValue[currentEditingLocale?.localeId]
-								: ''
-						}
-						editorConfig={editorConfig}
-						name={name}
-						onBlur={onBlur}
-						onChange={(content) => handleContentChange(content)}
-						onFocus={onFocus}
-						onSetData={(event) => {
-							const editor = event.editor;
+					{Liferay.FeatureFlags['LPD-11235'] ? (
+						<CKEditor5ClassicEditor
+							className="w-100"
+							config={{
+								...advancedClassicEditorConfig,
+							}}
+							data={
+								currentValue
+									? currentValue[
+											currentEditingLocale?.localeId
+										]
+									: ''
+							}
+							onChange={(event, editor) =>
+								handleContentChange(editor.getData())
```

Comment:
[onChange vs onchange:data](https://ckeditor.com/docs/ckeditor5/latest/api/module_engine_model_document-Document.html#event-change:data)?

Reply:
Nice catch on theAPI! I'll do some testing to see how the variants work in `onChange`, and broader context.